exercises/practice/fizz-buzz/zcl_fizz_buzz.clas.testclasses.abap

```diff
@@ -0,0 +1,41 @@
+class ltcl_fizz_buzz DEFINITION FOR TESTING RISK LEVEL HARMLESS DURATION SHORT FINAL.
+
+  private section.
+    data cut type ref to zcl_Fizz_Buzz.
+
+    METHODS setup.
+    methods test_fizz_Buzz FOR TESTING RAISING cx_static_check.
+
+endclass.
+
+
+class ltcl_fizz_buzz implementation.
+
+  METHOD setup.
+    cut = NEW ZCL_FIZZ_BUZZ( ).
+  ENDMETHOD.
+
+  method test_fizz_Buzz.
+  data result TYPE string_tab11.
```

Comment:
need testcases here instead of the implementation

Reply:
The 'implementation' is there to compare the string tables with the correct solution.
Make more sense to me than creating the table manually